reestructuring README

VindeeR opened the pull request:
Working on commands to be generic and avoiding running into docker execution problems

VindeeR committed:
Update README.md

VindeeR committed:
Update README.md

VindeeR committed:
Update README.md

VindeeR committed:
Update README.md

manuel-g-castro commented on README.md:
```diff
@@ -35,16 +35,16 @@ Generate a public-private pair key to configure the passwordless connection to t
-docker build . -t mmarciani/slurm-openssh-container
-docker build --build-arg SLURM_TAG='slurm-23-02-7-1' --build-arg PUBLIC_KEY='ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIH8a5WpSERO2+dXt1mISa8oS2Yc7VkSzhy2OuFwqnohP mgimenez@bsces107930' -t mmarciani/slurm-openssh:23-02-7-1 .
+docker build . -t ${USER}/slurm-openssh:23-02-7-1
+docker build --build-arg SLURM_TAG='slurm-23-02-7-1' --build-arg PUBLIC_KEY="$(cat ~/.ssh/container_root_pubkey.pub)" -t ${USER}/slurm-openssh:23-02-7-1 .
```
About the `${USER}`, it might be a bit narcissistic, but people normally use the developer's name there. Like [here](https://hub.docker.com/r/giovtorres/docker-centos7-slurm), he named the image "dev'sname/nameoftheimage".  

We could maybe put it under [autosubmit](https://hub.docker.com/r/autosubmit/autosubmit)? Or BSC?

manuel-g-castro reviewed (commented):
Thank you soo much .

VindeeR committed:
Update README.md

adding a pointers given, and changing commands in order to become more dynamic changing SLURM VERSION

manuel-g-castro reviewed (approved):
Looks good!
